www/main/settings/components/SettingViewer.jsx

```diff
@@ -161,13 +162,13 @@ const SettingViewer = () => {
               हे भाई ! प्रभू के दास अपने प्रभू से जो कुछ माँगते हैं वह वही कुछ उनको देता है।
             </div>
           </div>
-          {buttonMarkup('translation', slideOrder, setSlideOrder)}
+          {/* {buttonMarkup('translation', slideOrder, setSlideOrder)} */}
```

Comment:
Veerji, do we have to have all the code as comment now.

Reply:
Yes, commented the import statement and here. It's not visible now.

Reply:
Okay